src/components/Input/type.ts

```diff
@@ -0,0 +1,18 @@
+import { Colors } from '@/styles/colors';
+
+export interface TextInputProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'onChange'> {
+  label: string;
+  onChange?: (value: string) => void;
+  value?: string;
+  placeholder?: string;
+  format?: (value: string) => string;
+  textAlign?: 'left' | 'center' | 'right';
+  width?: string;
+  fontColor?: Colors;
+  type?: 'text' | 'password';
+}
+
+export interface LineInputProps extends TextInputProps {
+  lineColor?: Colors;
+}
```

Comment:
TextInputProps 는 나중에 만들어질 TextInput 들의 기본적인 타입이라고 생각하고 type.ts 에 넣은 것 같아요! 그 부분은 그대로 type.ts 파일에 넣고 LineInputProps 만 컴포넌트 와 같이 두었습니다! [3b18374](https://github.com/next-step/react-payments/pull/75/commits/3b18374f534c17edaa459090125ce2688ff6e798)
Merge state
Add more commits by pushing to the [main](https://github.com/eamon3481/react-payments) branch on [eamon3481/react-payments](https://github.com/eamon3481/react-payments).

Changes requested
1 review requesting changes 
1 change requested
This branch has no conflicts with the base branch
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.
@eamon3481

 
Add heading textAdd bold text, <Cmd+b>Add italic text, <Cmd+i>
Add a quote, <Cmd+Shift+.>Add code, <Cmd+e>Add a link, <Cmd+k>
Add a bulleted list, <Cmd+Shift+8>Add a numbered list, <Cmd+Shift+7>Add a task list, <Cmd+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply
Leave a comment
선택된 파일 없음
Attach files by dragging & dropping, selecting or pasting them.
Styling with Markdown is supported
Remember, contributions to this repository should follow our [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines).
 ProTip! Add [.patch](https://github.com/next-step/react-payments/pull/75.patch) or [.diff](https://github.com/next-step/react-payments/pull/75.diff) to the end of URLs for Git’s plaintext views.
Reviewers
@[jho2301](https://github.com/jho2301)
jho2301
jho2301 requested changes
Re-request review
Still in progress?
Assignees
@[jho2301](https://github.com/jho2301)
jho2301
Labels
None yet
Projects
None yet
Milestone
No milestone
Development
Successfully merging this pull request may close these issues.

None yet

Notifications
Customize
You’re receiving notifications because you authored the thread.
2 participants
@eamon3481
@jho2301
Allow edits by maintainers